<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Automaticpage</class>
 <widget class="QWidget" name="Automaticpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="Homebutton2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="consigne">
       <property name="styleSheet">
        <string notr="true">font: 40 italic 12pt &quot;DejaVu Sans&quot;;
color: rgb(204, 0, 0);</string>
       </property>
       <property name="text">
        <string>Note accepted : c0 until c9 (notes must be sperated by space)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(225, 225, 225);
font: 14pt &quot;DejaVu Sans&quot;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(225, 225, 225);
font: 12pt &quot;DejaVu Sans&quot;</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="errorMessage">
       <property name="styleSheet">
        <string notr="true">font: 40 italic 12pt &quot;DejaVu Sans&quot;;
color: rgb(204, 0, 0);</string>
       </property>
       <property name="text">
        <string>there is an invalid note in your text zone</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
